types/types.ts

```diff
@@ -18,3 +25,28 @@ export interface ArticlesResponse {
   list: Article[];
   totalCount: number;
 }
+
+export interface CommentsResponse {
+  list: Comment[];
+  nextCursor: null;
+}
+
+export interface Comment {
+  id: number;
+  content: string;
+  createdAt: string;
+  updatedAt: string;
+  writer: CommentWriter;
+}
+
+export interface CommentWriter {
+  id: number;
+  nickname: string;
+  image: string;
+}
```

Comment:
```suggestion
export interface Writer {
  id: number;
  nickname: string;
  }

export interface CommentWriter extends Writer{
  image: string;
}
```

비슷한 구조를 가진 간단한 interface끼리 확장해서 쓸 수 있어요